src/handlers/notebook.ts

```diff
@@ -53,21 +59,26 @@ export class NotebookHandler implements IDisposable {
       return;
     }
     this.isDisposed = true;
+    this.cleanAllCells();
     this.cellManager.dispose();
     Signal.clearData(this);
   }
 
+  protected cleanAllCells() {
```

Comment:
There is also some cleanup happening a bit below in the `onCurrentFrameChanged` method:

```typescript
cells.forEach(cell => CellManager.cleanupHighlight(cell));
```

Should we try to group that together?

Reply:
I make new exported function now for setOff line numbers.